src/openvic-simulation/misc/Define.cpp

```diff
@@ -150,15 +150,25 @@ bool DefineManager::load_define_years(Timespan& value, Define::Type type, std::s
 DefineManager::DefineManager()
   : // Date
 	start_date { 1836, 1, 1 },
-	end_date { 1936, 1, 1 }
+	end_date { 1936, 1, 1 },
 
 	// Country
+	great_power_rank { 8 },
```

Comment:
Why init the defines when you must overwrite this value with the value read from file?
If the value is missing from the files, we shouldn't fall back to some hardcoded value but instead error.

Reply:
It's best practice to initialise all variables so nothing contains random data, and the current system will log an error if the define is missing but making the game not start due to load errors is a larger change beyond this PR.